bundles/org.openhab.binding.boschshc/README.md

```diff
@@ -12,15 +12,24 @@ Binding for the Bosch Smart Home Controller:
 
 ## Limitations
 
- - Initial connection to Bosch SHC is not yet implemented (I bootstrapped the system using external software)
- - Discovery
+ - Discovery of Things
+ - Discovery of Bridge
 
 ## Discovery
 
-Not yet implemented. Configuration via configuration files.
+Not yet implemented. Configuration via configuration files or Paper UI.
 
 ## Binding Configuration
 
+You need to provide at least the ipAddress and the system password of your Bosch Smart Home Controller
+and a path to a JKS keystore file location.
+At this location a keystore file with a self signed certificate is created and stored if not present at the first setup.
+This certificate is used for pairing between the Bridge and the Bosch SHC. 
+
+*Press the Bosch Smart Home Controller Bridge button until the LED starts blinking after you save your settings for pairing*.
```

Comment:
```suggestion
*Press the Bosch Smart Home Controller Bridge button until the LED starts blinking after you save your settings for pairing. If you don't manage to pair the device within four minutes, you will have to restart OpenHab to trigger another connection attempt.*
```

Reply:
Currently three minutes are the limit and from my point of view a new pairing is started if you edit a Bosch SHC PaperUI setting and save it, because a new Bridge instance is created.